service.subtitles.supersubtitles/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.subtitles.supersubtitles"
+       name="Super Subtitles"
+       version="0.0.23"
+       provider-name="fape">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="xbmc.addon" version="17.9.910"/>
+    <import addon="script.module.kodi-six" version="0.1.0"/>
+    <import addon="vfs.libarchive" version="1.0.5"/>
+  </requires>
+  <extension point="xbmc.subtitle.module"
+             library="service.py" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Super Subtitles</summary>
+    <description lang="en_GB">Search and download subtitles from Super Subtitles (feliratok.info)</description>
+    <platform>all</platform>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
It'd be nice to have an spdx identifier instead: `GPL-2.0`

Reply:
there are 10 addons in leai branch such a long license description

Reply:
Again: [official documentation](https://kodi.wiki/view/Addon.xml#.3Clicense.3E)

> <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>

Reply:
**it would be nice** != Do it